ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -10,11 +10,54 @@ import LlamaKit
 import Nimble
 import Quick
 import ReactiveCocoa
+import Foundation
+
+private func startSignalProducer<T: Equatable, E: Equatable>(signalProducer: SignalProducer<T, E>, #expectSentValue: T?, sentError expectSentError: E?, #complete: Bool) {
+	startSignalProducer(signalProducer, expectSentValues: expectSentValue.map { [$0] } ?? [], sentError: expectSentError, complete: complete)
+}
+
+private func startSignalProducer<T: Equatable, E: Equatable>(signalProducer: SignalProducer<T, E>, expectSentValues: [T] = [], sentError expectSentError: E?, #complete: Bool) {
```

Comment:
I would like to avoid a helper like this. If we _really_ want to make it more convenient, I'd favor writing one or more Nimble matchers instead.